docs/guide/docs/quickstart.md

```diff
@@ -3,84 +3,100 @@ id: quickstart
+You can get additional information from the emulator's debugger by selecting the debug button. This panel is specifically designed for you, as the bot owner, to understand why you get a specific answer. Note that it is only available for authenticated users. It includes all sorts of useful information: the dialog engine's elected suggestion, nodes flowed through, and natural language intents or questions. You can also view the raw JSON event that contains all details if you need further data.
 
 ![Emulator Hello](assets/debugger.png)
 
 ### Question and Answer
 
-One of the modules shipped by default with Botpress is the Q&A (Question and Answer) module. This module allows you to easily and quickly add knowledge to your bot without having to create a flow or code anything. Q&A uses the NLU engine to detect the questions and automatically answer them.
+One of the modules shipped by default with Botpress is the Q&A (Question and Answer) module. This module allows you to easily and quickly add knowledge to your bot without creating a flow or writing code. Q&A uses the NLU engine to detect the questions and automatically answer them.
 
 #### Adding a Q&A
 
 1. Click the **Q&A** panel on the left.
 2. Click the "**Add New**" button.
 3. Type in the "Questions" textbox any question, for example: "How old are you?". Then add as many different ways of asking the same question as you want.
 4. In the "Answers" section, type the message you want your bot to provide, then press Enter to commit the answer.
-5. Click "**Add**".
+5. Click "**Add**."
 
-The Q&A module will automatically retrain the NLU model in the background and tell you when it's completed, indicated in the bottom right. You can speak again with your bot and it should respond to your question correctly, even if you make some typos!
+The Q&A module will automatically retrain the NLU model in the background and tell you when it's completed, indicated in the bottom right. You can speak again with your bot, and it should respond to your question correctly, even if you make some typos!
 
-##### Q&A Example
+#### Q&A Example
 
 ![A new Q&A](assets/newqna.png)
 
 ## Share your bot
 
-#### Standalone chat
+### Standalone chat
+
+Congratulations on building your first bot! You can share this bot with your friends, provided they have access to your computer. It is available at `http://localhost:3000/s/<<Bot ID>>`.
+
+### Website embedding
+
+You can also embed this bot on a website. To learn how, check out the example provided in `assets/modules/channel-web/examples/embedded-webchat.html`, which you can see by browsing [**here**](http://localhost:3000/assets/modules/channel-web/examples/embedded-webchat.html).
```

Comment:
There is also a full doc section on this, please link to it

Reply:
Linked